tests/testthat/test_query.R

```diff
@@ -33,17 +33,15 @@ expect_na <- function(object) {
 test_that("stops when handle is null", {
   expect_error(results <-
                  query(NULL, "SELECT * FROM timeseries IN RANGE(2017, +1y)")
-               ,
-               regexp = "type=NULL")
+               , regexp = "type=NULL")
```

Comment:
I hope you use the automatic formatter :).

Reply:
It was automatically formatted.